studio.py

```diff
@@ -30,14 +235,24 @@
     help="Proxy Service to use.",
 )
 @click.option("--record/--replay", default=False)
-@click.option("--certutil", help="Path to certutil.")
+@click.option(
+    "--certutil", help="Path to certutil. Note: Only when recording and Only on Android"
+)
+@click.option(
+    "--sites",
+    help="Json file containing the websites information we want ro record. Note: Only when recording",
+)
+@click.option(
+    "--path", default="Recordings", help="Path where to store/are stored the recordings"
```

Comment:
Why is this optional now?

Reply:
You resolved this but I don't see a response. What happens if you don't specify a path?